src/pages/komodo-defi-framework/api/legacy/setprice/index.mdx

```diff
@@ -15,20 +15,20 @@ The `setprice` order is always considered a `sell`, for internal implementation
-| rel\_confs        | number                     | number of required blockchain confirmations for rel coin atomic swap transaction; default to rel coin configuration if not set                                                                                                                                                                                                |
-| rel\_nota         | bool                       | whether dPoW notarization is required for rel coin atomic swap transaction; default to rel coin configuration if not set                                                                                                                                                                                                      |
-| save\_in\_history | boolean                    | Defaults to `true`. If set to `false` no order history will be saved (though order status will be temporarily stored while in progress). If `true`, each order's short record history is stored in a local SQLite database table, and when the order is cancelled or fully matched, it's history will be saved as a json file |
+| Structure         | Type    | Description                                                                                                                                                                                                                                                                                                                   |
+| ----------------- | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| base              | string  | the name of the coin the user desires to sell                                                                                                                                                                                                                                                                                 |
+| rel               | string  | the name of the coin the user desires to receive                                                                                                                                                                                                                                                                              |
+| price             | various | the amount of `rel` the user wants per one unit of the `base` coin. Can be a numeric string, expressed using standard [RationalValue](/komodo-defi-framework/api/common_structures/#rational-value) or [FractionalValue](/komodo-defi-framework/api/common_structures/#fractional-value) objects.                             |
```

Comment:
> the amount of `rel` the user wants per one unit of the `base` coin.

I think its better not to skip the action word.
Here (in setprice) user is "willing to receive".
In Buy, as another example, in the price we use "willing to pay".
I mean these "pay/send" and "receive" are marker words.

Reply:
updated to `the amount of `rel` coins the user wants to receive for each unit of the `base` coin spent.`